procedures/MT/rotator/rotator_verification.ipynb

```diff
@@ -0,0 +1,1822 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        ramp_generator_1 = simactuators.RampGenerator([0.7], [1.5], [0.068], track_advance_time)
</pre><p>These values are from tekniker? Might be worth reminding the reader where these come from... or are these what you're trying to tune?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Explained in the commit: [30d19d5](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/30d19d514c64981689c466b89131d18ff78a1464). Thanks!